common/static/common/js/spec/discussion/view/discussion_topic_menu_view_spec.js

```diff
@@ -45,7 +50,18 @@
                                     'sort_key': null,
                                     'is_cohorted': false,
                                     'id': 'c49f0dfb8fc94c9c8d9999cc95190c56'
+                                },
+                                'Very long category name': {
```

Comment:
P.S. Did we find the category name that was actually causing the bug on edx101? Would be interesting to just run it through this test to make sure.